pkg/common/catalog/init.go

```diff
@@ -5,6 +5,8 @@ import (
 	"sort"
 	"sync"
 
+	"github.com/spiffe/spire/pkg/common/telemetry"
```

Comment:
This import is out of order in many places. Normally I'd turn a blind eye but there's about a dozen occurrences in this PR so I felt compelled to mention it

Reply:
oops :)